yt_dlp/extractor/biqle.py

```diff
@@ -32,53 +36,70 @@ class BIQLEIE(InfoExtractor):
             'id': '-44781847_168547604',
             'ext': 'mp4',
             'title': 'Ребенок в шоке от автоматической мойки',
+            'description': 'Ребенок в шоке от автоматической мойки — Смотреть онлайн',
             'timestamp': 1396633454,
-            'uploader': 'Dmitry Kotov',
             'upload_date': '20140404',
-            'uploader_id': '47850140',
+            'thumbnail': r're:https://[^/]+/c535507/u190034692/video/l_b84df002\.jpg',
         },
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
-        embed_url = self._proto_relative_url(self._search_regex(
-            r'<iframe.+?src="((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^"]+)".*?></iframe>',
-            webpage, 'embed url'))
+
+        title = self._search_regex(
+            r'<meta.*itemprop ?= ?"name".*content ?= ?"([^"]+)".*/>',
+            webpage, 'Title', default='Empty Title', fatal=False)
+        uploadDate = self._search_regex(
+            r'<meta.*itemprop ?= ?"uploadDate".*content ?= ?"([^"]+)".*/?>',
+            webpage, 'Upload Date', fatal=False)
+        timestamp = unified_timestamp(uploadDate)
+        description = self._search_regex(
+            r'<meta.*itemprop ?= ?"description".*content ?= ?"([^"]+)".*/>',
+            webpage, 'Description', fatal=False)
+
+        globalEmbed_url = self._search_regex(
+            r'<script.+?window.globEmbedUrl = \'((?:https?:)?//(?:daxab\.com|dxb\.to|[^/]+/player)/[^\']+)\'.*?></script>',
+            webpage, 'global Embed url', flags=re.DOTALL)
```

Comment:
dont think dotall is needed

Reply:
All 3 dotall are required.
If I remove any of them the script returns error.

![image](https://user-images.githubusercontent.com/216170/153737975-63ab9cd9-3de4-4cad-9ef9-bf7bece63585.png)
![image](https://user-images.githubusercontent.com/216170/153737984-9c34ce72-58cc-4d81-91b6-264ff06d1d34.png)
![image](https://user-images.githubusercontent.com/216170/153737994-dc9c48b9-e62b-4589-b802-1d26520d4171.png)